src/preact.d.ts

```diff
@@ -277,8 +279,8 @@ declare namespace JSX {
 		charSet?:string;
 		challenge?:string;
 		checked?:boolean;
-		class?:string;
-		className?:string;
+		class?:string | Object;
+		className?:string | Object;
```

Comment:
would this be better `string | {[key:string]:string}`?

all values and keys will then be forced as string